shuup/front/checkout/addresses.py

```diff
@@ -133,6 +133,12 @@ def form_valid(self, form):
             self.storage["company"] = form.forms["company"].save(commit=False)
         else:
             self.storage["company"] = None
+
+        process_and_save_basket = any([form[form_key].has_changed() for form_key in form.forms])
+        if process_and_save_basket:
+            self.process()
+            self.basket.save()
+            self.basket.storage.add_log_entry(self.basket, _("Saved addresses."))
```

Comment:
instead of calling the storage, why can't we create a method in BaseBasket to log some event? By default we could just add some line to `basket._data["logs"]` ?